src/utils.hpp

```diff
@@ -0,0 +1,51 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2019 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef CSP_WMS_UTILS_HPP
+#define CSP_WMS_UTILS_HPP
+
+#include "../../../src/cs-utils/convert.hpp"
+
+#include <optional>
+#include <regex>
+
+namespace csp::simpleWmsBodies {
+
+/// Struct of timeintervalls of the data set
+struct timeInterval {
+  boost::posix_time::ptime startTime;
+  boost::posix_time::ptime endTime;
+  std::string              mFormat;
+  int                      mIntervalDuration;
+};
+
+/// A single WMS data set.
+struct Wms {
```

Comment:
I would prefer a more descriptive name here.

```suggestion
struct WMSConfig {
```